spec/presenters/activities_presenter_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "rails_helper"
+
+describe ::ActivitiesPresenter do
+  subject(:activities) { described_class.new(params) }
+  let(:params) { {} }
+  let(:user) { FactoryGirl.create(:user) }
+  let(:second_user) { FactoryGirl.create(:user) }
+  let(:device) { FactoryGirl.create(:device) }
+  let(:second_device) { FactoryGirl.create(:device) }
+  let!(:activity) { FactoryGirl.create(:activity, owner_id: user.id, trackable_type: "Approval") }
+  let!(:second_activity) do
+    FactoryGirl.create(:activity, owner_id: second_user.id, trackable_type: "Device", trackable_id: device.id)
```

Comment:
Since `FactoryGirl::Syntax::Methods` is included in rails helper for testing, we can omit 'FactoryGirl' and write just `create(...)`